packages/earljs/test/validators/smartEq.test.ts

```diff
@@ -161,7 +161,8 @@ describe('smartEq', () => {
   describe('plugin', () => {
     let smartEq: SmartEqType
     let loadSmartEqRules: LoadSmartEqRulesType
-    beforeEach(() => {
+    beforeEach(function () {
+      this.timeout(5_000)
```

Comment:
why do we need these timeouts?

Reply:
For reasons unknown to me the CI on macos failed the beforeEach with a timeout after 2s